src/views/validated-designs/guide/detail-view.module.css

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+.linkBoxWrapper {
+	--gap: 24px;
+	--item-count: 2;
```

Comment:
IMO, I don't think having variables here is worth it for 1-2 places and it's easier read the pixels
```suggestion
```